include/zephyr/bluetooth/audio/has.h

```diff
@@ -131,6 +131,26 @@ int bt_has_client_discover(struct bt_conn *conn);
  */
 int bt_has_client_conn_get(const struct bt_has *has, struct bt_conn **conn);
 
+/** @brief Preset operations structure. */
+struct bt_has_preset_ops {
```

Comment:
Are you expecting to add more callbacks to this struct?

Reply:
Yes, at least one